taipy/gui_core/viselements.json

```diff
@@ -430,22 +430,28 @@
                         "doc": "If False, the data node expiration date is not visible."
                     },
                     {
-                        "name": "show_properties",
+                        "name": "show_custom_properties",
                         "type": "bool",
                         "default_value": "True",
                         "doc": "If False, the data node properties are not visible."
                     },
+                    {
```

Comment:
Please move that just before show_history.
The order should be:
- "data_node"
- "show_data"
- "show_properties"
- "show_history"
- "active"
- "expandable"
- "expanded"
- "show_config"
- "show_owner"
- "show_owner_label"
- "show_custom_properties"
- "show_edit_date"
- "show_expiration_date"
- "chart_config"
- "scenario"